plugins/Toolbox/src/Toolbox.py

```diff
@@ -169,6 +169,27 @@ def _onAppInitialized(self) -> None:
             "materials_showcase": QUrl("{base_url}/showcase".format(base_url=self._api_url))
         }
 
+    OLD_PLUGINS = ['3DPrinterOS',
+                   'BarbarianPlugin',
+                   'CuraSolidWorksPlugin',
+                   'MakePrintablePlugin',
+                   'OctoPrintPlugin',
+                   'OrientationPlugin',
+                   'ZOffsetPlugin',
+                   'cura-siemensnx-plugin']
+
+    # check for plugins that were installed with the old plugin-browser
+    def _isOldPlugin(self, plugin_id: str) -> bool:
+        if plugin_id in self.OLD_PLUGINS and plugin_id in self._plugin_registry.getInstalledPlugins():
```

Comment:
I'm going to change this so that like the other upgrading code, the "update available" designation is made based on the call from the server.

The new logic will look like: "for each plugin from the server, if installed in plugin registry and not installed in package manager, _isOldPlugin = true"